assets/backport/go.mod

```diff
@@ -23,5 +23,5 @@ require (
 	golang.org/x/term v0.15.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/protobuf v1.25.0 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
+	gopkg.in/yaml.v3 v3.0.0 // indirect
```

Comment:
Diff for those looking for it: https://github.com/go-yaml/yaml/compare/9f266ea9e77c4c7aab4cf02650570e7c7b3031a5...v3.0.0.